keras_cv/models/__init__.py

```diff
@@ -43,6 +43,18 @@
 from keras_cv.models.backbones.densenet.densenet_backbone import (
     DenseNetBackbone,
 )
+from keras_cv.models.backbones.detectron2.detectron2_aliases import (
```

Comment:
Afaik, the backbone is basically the same as the official ViTDet, so there may not be a need to call it a SAM{name}Backbone

Reply:
Sounds good, thanks for looking into it!

Reply:
Done.